data/transactions/logic/evalAppTxn_test.go

```diff
@@ -1761,9 +1761,7 @@ int 1
 `
 
 	for _, unified := range []bool{true, false} {
-		t.Run(fmt.Sprintf("unified=%t", unified), func(t *testing.T) {
-			// t.Parallel() NO! unified variable is actually shared
-
+		t.Run(fmt.Sprintf("unified=%t", unified), func(t *testing.T) { //nolint:paralleltest // NO t.Parallel(). unified variable is actually shared
```

Comment:
Does this linter actually complain about every t.Run too? Or just top-level `func Test*`?

Reply:
Also `t.Run()`. For a full list of the errors it catches: https://github.com/kunwardeep/paralleltest#examples

Reply:
Nevermind, I see it does.